graphql/graphql.go

```diff
@@ -440,9 +440,13 @@ func (t *Transaction) AccessList(ctx context.Context) (*[]*AccessTuple, error) {
 	accessList := tx.AccessList()
 	ret := make([]*AccessTuple, 0, len(accessList))
 	for _, al := range accessList {
+		var sKeys []common.Hash
```

Comment:
`al := al` would also fix it

Reply:
Would be even better to change the type of `AccessTuple.storageKeys` to non-pointer.

Reply:
fixed, PTAL